main.tf

```diff
@@ -5,12 +5,59 @@ resource "aws_instance" "web" {
   tags = {
     Name = "web"
   }
+
+  metadata_options {
+    http_endpoint = "disabled"
+    http_tokens   = "required"
+  }
 }
 
 resource "aws_s3_bucket" "bucket" {
   bucket = "my-tf-test-bucket"
 
   tags = {
-    Name        = "bucket"
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure AWS S3 Buckets are accessible to all authenticated users**
 
**Vulnerability**
S3 bucket Access is allowed to all AWS Account Users.
 
**Remediation**
Amazon S3 Bucket ACL with full control permission to authenticated users allows anyone with an AWS account to access objects in the bucket. When read and write access is granted to authenticated users, they can read, edit and delete the objects in the bucket. It is a recommended practice to remove full control permission from S3 Bucket ACL.

source: accurics-stageone